### ISSUE TITLE
Polders generation height to low? 

### ISSUE THREAD

**Issue by Blaueschmied** [zmatez/wildnature#30]
Hey,

i have an issue to share with you!
i'm convinced that your Polders Biome is generating far to low on world generation.
it's initial block height is at 40 blocks and as you can see down below, the surrounding biomes cutting through this or vice versa. maybe to this point it isn't known or i just have overseen it.

![2019-12-24_10 01 26](https://user-images.githubusercontent.com/59196184/71404984-5b4ab900-2634-11ea-9aaa-b4c35ad465e8.png)

**Comment by BlackNightDragon** [comment]
Yeah, i have the same problem: https://github.com/zmatez/wildnature/issues/31

**Comment by ZippyMagician** [comment]
Known issue, we are fixing it. It’s caused by our custom river generation. It will be a smooth transition.
